packages/react/src/generators/application/lib/bundlers/add-rspack.ts

```diff
@@ -19,38 +18,11 @@ export async function initRspack(
   const { rspackInitGenerator } = ensurePackage('@nx/rspack', nxVersion);
   const rspackInitTask = await rspackInitGenerator(tree, {
     ...options,
-    addPlugin: false,
     skipFormat: true,
   });
   tasks.push(rspackInitTask);
 }
 
-export async function setupRspackConfiguration(
-  tree: Tree,
-  options: NormalizedSchema<Schema>,
-  tasks: any[]
-) {
-  const { configurationGenerator } = ensurePackage('@nx/rspack', nxVersion);
-  const rspackTask = await configurationGenerator(tree, {
-    project: options.projectName,
-    main: joinPathFragments(
-      options.appProjectRoot,
-      maybeJs(
-        {
-          js: options.js,
-          useJsx: true,
-        },
-        `src/main.tsx`
-      )
-    ),
-    tsConfig: joinPathFragments(options.appProjectRoot, 'tsconfig.app.json'),
-    target: 'web',
-    newProject: true,
-    framework: 'react',
-  });
-  tasks.push(rspackTask);
-}
-
```

Comment:
If someone decides they dont want crystal - or their workspace already has `useInferencePlugins: false` what would happen with this missing?

Reply:
It will behave the same as Webpack. I tested this everything should be working.